app/models/shipit/deploy_spec/file_system.rb

```diff
@@ -136,6 +137,20 @@ def app_name
         @app_name ||= Shipit.app_name.downcase
       end
 
+      SHIPIT_CONFIG_INHERIT_FROM_KEY = "inherit_from"
+      def build_config(path, config_obj)
+        return config_obj if config_obj.blank? || !config_obj.key?(SHIPIT_CONFIG_INHERIT_FROM_KEY)
+
+        inherits_from_path = path.dirname.join(config_obj.delete(SHIPIT_CONFIG_INHERIT_FROM_KEY))
+        if inherits_from_path.exist?
```

Comment:
Open question: does a non existing path should noop or result in an error?

The problem is kinda always the same though, we don't have a good way of reporting `shipit.yml` errors, so perhaps nooping is the reasonable thing to do.